File: algorithm/boj/_2000/2562.java
package TIL.algorithm.boj._2000;

import java.io.*;
import java.util.Arrays;

class Main2562 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int idx = 0;
        
        int[] arr = new int[9];
        for(int i=0; i<arr.length; i++) {
            arr[i] = Integer.parseInt(br.readLine());
        }
        
        int[] sort_arr = arr.clone();
        Arrays.sort(sort_arr);
        int max_value = sort_arr[(arr.length)-1];
        
        for(int i=0; i<arr.length; i++) {
            if(arr[i] == max_value)
                idx = i + 1;
        }
        
        bw.write(max_value+"\n");
        bw.write(idx+"\n");
        br.close();
        bw.flush();
        bw.close();
    }
}

File: algorithm/boj/_2000/2562.py
import sys

if __name__ == "__main__":
    li = []
    
    for i in range(9):
        li.append(int(sys.stdin.readline()))
    
    sort_list = sorted(li)
    max_value = sort_list[-1]
    
    for i in range(len(li)):
        if li[i] == max_value:
            idx = i
    
    print(max_value)
    print(idx+1)